.github/workflows/publish.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish to ECR
+on:
+  workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v3
+    
+          - name: Set up QEMU
+            uses: docker/setup-qemu-action@v2
+    
+          - name: Set up Docker Buildx
+            uses: docker/setup-buildx-action@v2
+        
+          - name: Configure AWS Credentials
+            uses: aws-actions/configure-aws-credentials@v2
+            with:
+              aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+              aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+              aws-region: ${{ secrets.AWS_REGION }}
+    
+          - name: Login to Amazon ECR
+            id: login-ecr
+            uses: aws-actions/amazon-ecr-login@v1
+
+          - name: Calculate timestamp tag
+            id: timestamptagcalc
+            run: |
+              ts=$(date +%s)
+              echo $ts
+              echo "tag=$ts" >> $GITHUB_OUTPUT
+    
+          - name: Build and push images to ECR
+            uses: docker/build-push-action@v4
+            with:
+              context: server
+              platforms: linux/amd64,linux/arm64
+              push: true
+              tags: ${{steps.login-ecr.outputs.registry}}/${{github.repository}}:timestamp-${{ steps.timestamptagcalc.outputs.tag }}
```

Comment:
oh you need the other tags too!

i just meant the last tag had to be the timestamp tag